src/data/User.ts

```diff
@@ -0,0 +1,10 @@
+interface Name {
```

Comment:
In practice prefix the name of the interface with the letter "I" to indicate that it's an interface. 
```suggestion
interface IName {
```